packit_service/worker/jobs.py

```diff
@@ -323,7 +323,10 @@ def _clean_workplace(self):
             logger.info("volume was not empty")
             logger.debug("content of the volume: %s" % globz)
         for item in globz:
-            shutil.rmtree(item)
+            if item.is_file():
```

Comment:
What about if the file is a symlink?

Reply:
Sorry I did not read documentation properly (https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file). Symbolic link is counted.